src/Raven.Client/ServerWide/Sharding/AddDatabaseShardOperation.cs

```diff
@@ -16,27 +16,29 @@ internal sealed class AddDatabaseShardOperation : IServerOperation<AddDatabaseSh
         private readonly int? _shardNumber;
         private readonly string[] _nodes;
         private readonly int? _replicationFactor;
+        private readonly bool? _dynamicNodeDistribution;
```

Comment:
this operation should be internal or public?

Reply:
Should be public. Fixed